### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by sunniiiw**

### CODE REVIEW

File: week5/sunny/problems/index.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+const isString = string => {
+  if (typeof string == "string") {
```
Comment (mxdi9i7) [inline week5/sunny/problems/index.js]: using triple equal

File: week5/sunny/problems/index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    return true;
+  } else {
```
Comment (mxdi9i7) [inline week5/sunny/problems/index.js]: you don't need the else

File: week5/sunny/problems/index.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    return false;
+  }
+};
+
+console.log(isString("future sphere"));
+console.log(isString(35));
+
+const isBlank = blank => {
+  if (blank.length == 0) {
```
Comment (mxdi9i7) [inline week5/sunny/problems/index.js]: triple equal

File: week5/sunny/problems/index.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    return true;
+  } else {
```
Comment (mxdi9i7) [inline week5/sunny/problems/index.js]: no else

File: week5/sunny/problems/index.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+const noSpace = str => {
+  if (str[0] === " ") {
+    str.splice(0, 1);
+  }
+  if (str[str.length - 1] === "") {
+    str.splice(str.length - 1, 1);
+  }
+  return str;
+};
+
```
Comment (mxdi9i7) [inline week5/sunny/problems/index.js]: missing phone number format

File: week5/sunny/qzone1/index.css [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+}
+
+.login-container {
+  position: absolute;
+  top: 95px;
+  width: inherit;
+}
+
+.login-container img {
+  width: 251px;
```
Comment (mxdi9i7) [inline week5/sunny/qzone1/index.css:22]: do not use arbitrary number for width

File: week5/sunny/qzone1/index.css [hunk 7]
```diff
@@ -0,0 +1,266 @@
...
+  cursor: pointer;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+form {
+  width: 272px;
+  margin: 0 71px;
```
Comment (mxdi9i7) [inline week5/sunny/qzone1/index.css:70]: why not use 70 or 72??

File: week5/sunny/qzone1/index.css [hunk 8]
```diff
@@ -0,0 +1,266 @@
...
+}
+
+input {
+  width: inherit;
+  height: 42px;
+  margin-bottom: 13px;
```
Comment (mxdi9i7) [inline week5/sunny/qzone1/index.css:76]: 13?

File: week5/sunny/qzone1/index.html [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+              >，防止盗号。
+            </p>
+          </div>
+          <form>
+            <input
+              type="email,tel,qq id"
+              placeholder="支持QQ号/邮箱/手机号登录"
+            />
+            <input type="password" placeholder="密码" />
+            <button><a href="#">登 录</a></button>
```
Comment (mxdi9i7) [inline week5/sunny/qzone1/index.html:54]: do not use a single space, use &nbsp; instead

**Review by mxdi9i7 — CHANGES_REQUESTED** [review]
missing taobao code

File: week5/sunny/qzone1/index.css (hunk 7 above)
Comment (sunniiiw) [reply week5/sunny/qzone1/index.css:70]: by following the data from inspect

File: week5/sunny/qzone1/index.css (hunk 8 above)
Comment (sunniiiw) [reply week5/sunny/qzone1/index.css:76]: which means i dont need to follow exactly the numbers shows in inspect. i could use 12 or 14 instead?

File: week5/sunny/qzone1/index.css (hunk 7 above)
Comment (mxdi9i7) [reply week5/sunny/qzone1/index.css:70]: always use an even number

File: week5/sunny/qzone1/index.css (hunk 8 above)
Comment (mxdi9i7) [reply week5/sunny/qzone1/index.css:76]: yes, always use even numbers

File: week5/sunny/qzone1/index.html (hunk 9 above)
Comment (mxdi9i7) [reply week5/sunny/qzone1/index.html:54]: `&nbsp;`

**Review by mxdi9i7 — APPROVED** [review]

**Pull request merged**